orb/orb.yml

```
@@ -315,11 +315,11 @@ commands:
             KUBE_VERSION=$(kubectl version -o yaml | yq .clientVersion.minor)
```

Should this line be like this?
```suggestion
            KUBE_VERSION=$(kubectl version -o yaml | yq eval .clientVersion.minor -)
```

---

And I guess it's the same with this one - we don't need to add the eval